### PR TITLE
{phys}[foss/2022a] CASTEP v22.11

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node311.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/smoors/425731679710f6dfb8df9a7b8d161cce for a full test report.

File: easybuild/easyconfigs/c/CASTEP/CASTEP-22.11-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+description = """
+CASTEP is an electronic structure materials modelling code based on density
+functional theory (DFT), with functionality including geometry optimization
+molecular dynamics, phonons, NMR chemical shifts and much more.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+# CASTEP is proprietary software, available under a free-of-charge license for academic use only.
+# Visit http://www.castep.org and navigate to "Getting Castep" to apply for a license.
```
Comment (branfosj) [inline easybuild/easyconfigs/c/CASTEP/CASTEP-22.11-foss-2022a.eb:16]: download_instructions = """CASTEP is proprietary software, available under a free-of-charge license for academic use only.
Visit http://www.castep.org and navigate to "Getting Castep" to apply for a license."""

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/CASTEP/CASTEP-22.11-foss-2022a.eb (hunk 1 above)
Comment (branfosj) [inline easybuild/easyconfigs/c/CASTEP/CASTEP-22.11-foss-2022a.eb:16]:
```suggestion
download_instructions = """CASTEP is proprietary software, available under a free-of-charge license for academic use
only. Visit http://www.castep.org and navigate to "Getting Castep" to apply for a license."""
```

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/03918d408762ccd2f736343ab00e1eff for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node302.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/smoors/e13f0e372ec2f3de94fdaaf58326f87b for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/50879a292a2ebb353c17e42b8302f3cc for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @smoors!

**Pull request merged**